### PR TITLE
Automatic GC

### PR DESCRIPTION
**Pull request by killerswin2**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    Implements a system by tracking last gc time, to run a gc if the threshold is passed.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

**Review by Bob-Murphy — APPROVED** [review]
lgtm

**Comment by Bob-Murphy** [comment]
@CalebSerafin you self-requested a review.
Please do so.
Cheers

File: A3A/addons/core/functions/Base/fn_garbageCleanerTracker.sqf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    // if the gc time goes above the threshold then it's time to gc.
+    if ((serverTime-A3A_lastGarbageCleanTime) > A3A_GCThreshold) then 
+    {
+        [] call A3A_fnc_garbageCleaner;
+        [localize "STR_A3A_GCTracker_tracker_title", localize "STR_A3A_GCTracker_tracker_ran_gc"] remoteExec ["A3A_fnc_customHint", 0];
+        Info("Garbage Cleaner Tracker has ran a gc as players reached threshold");
+    };
+
+    // sleep 5 minutes then check again
+    sleep 300;
```
Comment (CalebSerafin) [inline A3A/addons/core/functions/Base/fn_garbageCleanerTracker.sqf:44]: You could just `sleep (A3A_GCThreshold / 4)` as there are no other commands to execute.

Comment (CalebSerafin) [reply A3A/addons/core/functions/Base/fn_garbageCleanerTracker.sqf:44]: Or enforce have a minium time with `sleep (300 max (A3A_GCThreshold / 4))`. Although with your current param options, the shorted sleep would still be 900 seconds.

**Review by CalebSerafin — APPROVED** [review]
A comment about sleep time. Reject the review if you want to change it base on the suggestion.

**Pull request merged**